lib/ansible/modules/network/sda/sda_rest.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: sda_rest
+
+short_description: Direct access to the Cisco Software-Defined Access (SDA) controller
+
+version_added: "2.6"
+
+description:
+    - Enables management of the Cisco Software-Defined Access controller, more commonly called DNA Center

```

Comment:
All descriptions should end with a dot, except the short_description.

Reply:
Makes sense, updated.